python/core/auto_generated/qgsrelation.sip.in

```diff
@@ -45,10 +44,11 @@ changed.
 %End
 
 
-    static QgsRelation createFromXml( const QDomNode &node, QgsReadWriteContext &context );
+    static QgsRelation createFromXml( const QgsProject &project, const QDomNode &node, QgsReadWriteContext &context );
```

Comment:
I'm afraid this qualifies as an API break, you should deprecate the old methods and add your new API.